packages/squiggle-lang/src/rescript/Utility/Operation.res

```diff
@@ -150,7 +154,12 @@ module Scale = {
     | #Multiply => Ok(a *. b)
     | #Divide => divide(a, b)
     | #Power => power(a, b)
-    | #Logarithm => logarithm(a, b)
+    | #Logarithm =>
+      if a < MagicNumbers.Epsilon.seven {
+        Ok(0.0)
```

Comment:
why? This feels pretty hacky to me.

Reply:
Still thinking it through, but if one has an xyShape, this approximates "take the logarithm of the y points, unless they are pretty close to 0"

Reply:
It is pretty hacky.

Reply:
You could throw an error. Returning an incorrect result is particularly bad, because users won't be able to tell.

Reply:
What's the verdict on `Error(NegativeInfinityError)` vs. `Ok(0.0)`?